Utilities/runtests.py

```diff
@@ -27,11 +27,12 @@
 plat = ThisPlatform(qemu = args.qemu[0])
 
 print("Reading tests...")
+if args.debug:
```

Comment:
This means you have to launch the load_tests in a debugger too, right?  Maybe --debug_load_tests should be its own argument for the rare case that you want to do that?

Reply:
aye good point. removed the commit for now